gradle/plugins/common/src/main/kotlin/junitbuild.java-library-conventions.gradle.kts

```diff
@@ -18,39 +18,18 @@ plugins {
 }
 
 rewrite {
-	activeRecipe("org.openrewrite.java.RemoveUnusedImports")
-	activeRecipe("org.openrewrite.staticanalysis.EqualsAvoidsNull")
-	activeRecipe("org.openrewrite.staticanalysis.ModifierOrder")
-	activeRecipe("org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods")
-	exclusions.add("**RunnerWithCustomUniqueIdsAndDisplayNames.java")
-	exclusions.add("**SpockTestCaseWithUnrolledAndRegularFeatureMethods.groovy")
+	activeRecipe("org.junit.openrewrite.recipe.CodeCleanup")
+	configFile = file("../gradle/config/rewrite.yml")
```

Comment:
FYI. it was `../` idk why, as the other files use `gradle/` as root.



Still thanks for your suggestion. It helped a lot. @vlsi

<img width="627" height="99" alt="Image" src="https://github.com/user-attachments/assets/8e4170d9-48ed-43c0-9ee2-0bca239f3ae8" />